test/Fail/Issue5706.err

```diff
@@ -2,3 +2,11 @@ Issue5706.agda:32,3-6
 Set₁₈₄₄₆₇₄₄₀₇₃₇₀₉₅₅₁₆₁₆ is not less or equal than Set
 when checking that the type Set₁₈₄₄₆₇₄₄₀₇₃₇₀₉₅₅₁₆₁₅ of an argument
 to the constructor set fits in the sort Set of the datatype.
+Issue5706.agda:32,3-6
+Set₁₈₄₄₆₇₄₄₀₇₃₇₀₉₅₅₁₆₁₅ is not less or equal than Set
+when checking that the type X → SET of an argument to the
+constructor set fits in the sort Set of the datatype.
```

Comment:
Don't know why the error is duplicated in the golden values now.  I do not see this when running Agda interactively or on the command line.